test/track-info.js

```diff
@@ -41,5 +41,20 @@ describe('A Google Music instance playing no music', function () {
       expect(this.result).to.have.property('duration');
       expect(this.result.duration).to.be.a('number');
     });
+    describe('via API', function () {
+      browserUtils.execute(function getInfoFromApi () {
```

Comment:
We should keep function names consistent with the methods they are calling. Can we update this to:

```js
browserUtils.execute(function getSongInfoFromApi () { 
```